cmd/down.go

```diff
@@ -83,11 +91,27 @@ func Down() *cobra.Command {
 
 	cmd.Flags().StringVarP(&devPath, "file", "f", utils.DefaultManifest, "path to the manifest file")
 	cmd.Flags().BoolVarP(&rm, "volumes", "v", false, "remove persistent volume")
+	cmd.Flags().BoolVarP(&all, "all", "A", false, "deactivate all running dev containers")
```

Comment:
minor doubt: is this case sensitive? is there a reason for "A" and not "a" ?

Reply:
Yup it is case sensitive. I went with "A" instead of "a" because that is what's commonly used by other tools to represent "all". For example: `kubectl get pods -A`